packages/@datapull/atlassian-origin/index.js

```diff
@@ -21,95 +22,46 @@ class AtlassianOrigin {
     };
   }
 
+  // To list all Atlassian users in specific organisation,
+  // that have access to jira tool,
+  // we have to find all users that are members of 'jira-software-users'.
+  // For those who have an access to confluence,
+  // we have to retrieve members of 'confluence-users' group.
   async pullData(_) {
     if (this.service === 'jira') {
       console.log(`[Atlassian Origin] pulling ${this.service} user data for ${this.organisation} with batch size ${this.batchSize}`);
-      return await this.getJiraData();
+      return await this.getUsers('jira-software-users');
```

Comment:
So in the specific case you're working on, the jira-software-users group define wether users have jira access or not. Seeing this application is open sourced, is this definition something that is used globally in each jira distribution? Otherwise we would introduce specific logic for one specific configuration of Jira that does not work for others.